### PR TITLE
Update string counts in database

### PR DESCRIPTION
**Pull request by mawaruDOLL**

### CODE REVIEW

File: collections/incidents.coffee [hunk 1]
```diff
@@ -67,4 +67,13 @@ if Meteor.isServer
...
\ No newline at end of file
+        Incidents.insert(newIncident)
+
+    # Convert string case and death counts to integers
+    incidents = Incidents.find({$or: [{cases: {$type: "string"}}, {deaths: {$type: "string"}}]}).fetch()
+    for incident in incidents
+      if incident.cases
+        updateField = {cases: parseInt(incident.cases)}
+      else
+        updateField = {deaths: parseInt(incident.deaths)}
```
Comment (nathanathan) [inline collections/incidents.coffee]: The incident could be a "specify other" in this case.

Comment (nathanathan) [inline collections/incidents.coffee]: Also, I think some error handling is needed in case parseInt doesn't return a number.

**Comment by mawaruDOLL** [comment]
I accounted for parseInt() returning NaN by setting the specify field with the value that failed to parse.

File: collections/incidents.coffee [hunk 2]
```diff
@@ -67,4 +67,22 @@ if Meteor.isServer
...
+          mongoProjection = {$set: {specify: incident.cases}, $unset: {cases: ""}}
+        else
+          mongoProjection = {$set: {cases: parsed}}
+      else if incident.deaths
+        parsed = parseInt(incident.deaths)
+        if parsed.toString() is "NaN"
+          mongoProjection = {$set: {specify: incident.deaths}, $unset: {deaths: ""}}
+        else
+          mongoProjection = {$set: {deaths: parsed}}
+      if mongoProjection
```
Comment (nathanathan) [inline collections/incidents.coffee]: I think the mongoProjection variable form the last iteration might still be defined.

**Comment by nathanathan** [comment]
👍

**Pull request merged**